specificatie/geboorte.yaml

```diff
@@ -0,0 +1,59 @@
+openapi: 3.0.3
+info:
+  title: Geboorte definities
+  version: 2.0.0
+  contact: {}
+paths: {}
+components:
+  schemas:
+    Geboorte:
+      type: object
+      description: |
+        Gegevens over de geboorte.
+        * **datum** : datum waarop de persoon is geboren.
+        * **land** : land waar de persoon is geboren
+        * **plaats** : gemeente waar de persoon is geboren. Is de persoon geboren buiten Nederland, dan bevat het antwoord alleen een buitenlandse plaatsnaam of aanduiding.
+      properties:
+        datum:
+          $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+        land:
+          $ref: 'land.yaml#/components/schemas/AbstractLand'
+        plaats:
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel"
+        inOnderzoek:
+          $ref: "#/components/schemas/AbstractGeboorteInOnderzoek"
+    AbstractGeboorteInOnderzoek:
+      type: object
+      description: |
+        Geeft aan welke gegevens over de geboorte van de persoon in onderzoek zijn. Zie de [functionele specificaties](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.1.0/features/in_onderzoek.feature)
+      properties:
+        type:
```

Comment:
datumIngangOnderzoek ontbreekt hier (en in de andere AbstractXXXInOnderzoek varianten).

Kunnen niet alle AbstractXXXInOnderzoek varianten overerven uit een InOnderzoekBasis (of hoe je het ook wilt noemen) met datumIngangOnderzoek en type? Dan hoeft de specifiekere AbstractXXXInOnderzoek alleen de discriminatoir mapping toe te voegen

Reply:
Heb ik niet gedaan omdat ik in LO GBA ziet AanduidingGegevensInOnderzoekXXX niet overal hetzelfde is. Bij AanduidingGegevensInOnderzoekNationaliteit kom ik het volgende formaat tegen TXT5..305

Reply:
Zit in de afgeleiden. type datumIngangInOnderzoek verschilt tussen de GBA en de HC type

Reply:
ik denk dat je niet op de juiste plek hebt gekeken in het LO GBA voor de definitie van aanduiding gegevens in onderzoek. Zie pag. 326:
<img width="509" alt="image" src="https://user-images.githubusercontent.com/37143495/151943816-006dc795-87ec-406f-b8b0-d62085edb8cc.png">